packages/kbn-cases-components/src/tooltip/utils.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export const getTruncatedText = (text: string, truncateLength: number): string => {
+  if (truncateLength < 0 || text.length <= truncateLength) {
```

Comment:
I think we should return the text if the `truncateLength` equals zero because if I do `myText.slice(0, 0)` it will return an empty string. Do you think this is it the expected behavior and we should leave it as it is? I am not sure what is best 🙂.

Reply:
yeah makes sense to return `text` if `truncateLength` is 0